i18n/en.json

```diff
@@ -3833,18 +3833,36 @@
   "notify_here.question": "By using **@here** you are about to send notifications to up to **{totalMembers} people**. Are you sure you want to do this?",
   "notify_here.question_timezone": "By using **@here** you are about to send notifications to up to **{totalMembers} people** in **{timezones, number} {timezones, plural, one {timezone} other {timezones}}**. Are you sure you want to do this?",
   "numMembers": "{num, number} {num, plural, one {member} other {members}}",
-  "onboarding_checklist.completed_subtitle": "We hope Mattermost is more familiar now. Need more help?",
```

Comment:
I don't get why we are changing the ids, It seems that the strings are the same.
Won't we lose our already translated  ones this way?

Reply:
these aren’t old Id's, were created as part of growth spike so to keep naming inline changed them.